tests/ci_build/Dockerfile.rmm

```diff
@@ -11,6 +11,14 @@ RUN \
     apt-key adv --fetch-keys https://developer.download.nvidia.com/compute/cuda/repos/ubuntu1804/x86_64/3bf863cc.pub && \
     apt-get update && \
     apt-get install -y wget unzip bzip2 libgomp1 build-essential ninja-build git && \
+    # Install gRPC for Federated Learning
+    git clone -b v1.49.1 https://github.com/grpc/grpc.git && \
+    pushd grpc && \
+    git submodule update --init && \
+    cmake -S . -B build -GNinja && \
+    cmake --build build --target install && \
+    popd && \
+    rm -fr grpc && \
```

Comment:
It could be better to use `grpc-cpp` package from Conda, as we won't have to build from the source

Reply:
Tried this, but looks like the `libprotobuf` conda package doesn't install the cmake modules: https://github.com/conda-forge/libprotobuf-feedstock/issues/68